src/config.ts

```diff
@@ -109,6 +110,7 @@ const buildAppContext = (): AppConfig => {
     serverPort: process.env.PORT || '3000',
     basePath: process.env.BASE_PATH || '/',
     openApiPath: process.env.OPENAPI_PATH || '/api-docs',
+    isProduction: String(process.env.NODE_ENV).toLowerCase() === 'production',
```

Comment:
can we do this the opposite to force us to enable the endpoint instead of hoping it gets disabled?
```suggestion
    isProduction: String(process.env.NODE_ENV).toLowerCase() !== 'development',
```

Reply:
or change your test to `isDevelopment` as that will be the positive cause and accomplish the disabled by default.